### PR TITLE
2452 copy and paste too much text

### PR DESCRIPTION
**Pull request by camgnostic**
closes mercycorps/TolaActivity#2452

Best balance I could strike between boilerplate-for-days and obscure-to-unreadable.  Thoughts?

### CODE REVIEW

File: templates/indicators/indicator_form_common_js.html [hunk 1]
```diff
@@ -1014,6 +1014,39 @@
...
+     */
+    function getTextAreaLengthValidator(fieldName, length) {
+        function validateField(e) {
+            const thisLength = length;
+            const $field = $(`#id_${fieldName}`);
+            const $validation = $(`#validation_id_${fieldName}`);
+            const $div = $(`#div_id_${fieldName}`);
+            let errMsg = null;
+            if ($field.val().length > thisLength) {
+                errMsg = interpolate("{% trans 'Please enter fewer than %s characters.'|escapejs %}", [thisLength]);
```
Comment (sanjuroj) [inline templates/indicators/indicator_form_common_js.html:1031]: Add translator comments?

File: templates/indicators/indicator_form_common_js.html [hunk 2]
```diff
@@ -1014,6 +1014,39 @@
         }
     }
 
+    /*
+     * Higher-order function to automate validation internals for a length-validated textarea
+     * Takes the field name (i.e. "justification") and max length (i.e. 400) and returns a validator for
+     * that field, which when called adds invalid classes/markings/labels, and returns false if not valid
+     * called both on the keyup event listeners below, and as part of whole-form validation
+     */
+    function getTextAreaLengthValidator(fieldName, length) {
```
Comment (sanjuroj) [inline templates/indicators/indicator_form_common_js.html:1023]: This is a textbook example of a closure.

**Review by sanjuroj — CHANGES_REQUESTED** [review]
Just a missing translator comment to update.  This seems a bit tedious but less bad than any of the other options I can think of.

File: templates/indicators/indicator_form_common_js.html (hunk 2 above)
Comment (camgnostic) [reply templates/indicators/indicator_form_common_js.html:1023]: and I got there the way I always do - writing the loop without a closure and wondering why everything is validating the last field in the list...

**Pull request merged**